drive.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+	"github.com/tealeg/xlsx"
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/drive/v2"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"golang.org/x/oauth2"
+)
+
+var eventFilePath = environmentName + "/Event Schedule.xlsx"
+
+func downloadFile(d *drive.Service, t http.RoundTripper, f *drive.File) {
```

Comment:
Authorization için oauth2 kullandığım için abi.. Oauth2 paketinde struct tipindeki Transport RoundTripper interface'ini implemente ediyor. Yani ben oradan parametre olarak oluşturduğum Transport'u gönderiyorum.